js/src/lib/Details/Markdown.js

```diff
@@ -12,24 +17,31 @@ const renderAndEscapeMarkdown = ({ source, githubRepo, gitHead }) => {
 
   if (githubRepo) {
     const { user, project, path } = githubRepo;
-    renderer.image = function(href, title, text) {
-      return `<img src="${prefixURL(href, {
-        base: 'https://raw.githubusercontent.com',
+    const prefix = (href, base) =>
+      prefixURL(href, {
+        base,
         user,
         project,
         head: gitHead ? gitHead : 'master',
         path,
-      })}" title="${title}" alt="${text}"/>`;
-    };
+      });
 
-    renderer.link = function(href, title, text) {
-      return `<a href="${prefixURL(href, {
-        base: 'https://github.com',
-        user,
-        project,
-        head: gitHead ? `tree/${gitHead}` : 'tree/master',
-        path,
-      })}" title="${title}">${text}</a>`;
+    renderer.image = (href, title, text) =>
+      `<img src="${prefix(href, GITHUB.raw)}" title="${title}" alt="${text}"/>`;
+
+    renderer.link = (href, title, text) =>
+      `<a href="${prefix(href, GITHUB.main)}" title="${title}">${text}</a>`;
+
+    renderer.html = function(html) {
+      return html
+        .replace(
+          /src="([^"]*)/g,
+          (match, href) => `src="${prefix(href, GITHUB.raw)}`
+        )
+        .replace(
+          /href="([^"]*)/g,
```

Comment:
Doing two replacements with the exact same regex seems wasteful - Could you do both of these in a single `.replace` call, depending on the URL?